libse/ManagedBitmap.cs

```diff
@@ -17,9 +17,8 @@ public ManagedBitmap(string fileName)
             try
             {
                 byte[] buffer = new byte[1024];
-                MemoryStream fd = new MemoryStream();
-                Stream fs = File.OpenRead(fileName);
-                using (Stream csStream = new GZipStream(fs, CompressionMode.Decompress))
+                using (var fd = new MemoryStream())
+                using (Stream csStream = new GZipStream(File.OpenRead(fileName), CompressionMode.Decompress))
                 {
                     int nRead;
                     while ((nRead = csStream.Read(buffer, 0, buffer.Length)) > 0)
```

Comment:
Yes, `CopyTo` method was added in .NET Framework 4.
`CopyToAsync` was added in 4.5.